DxDispatch/src/dxdispatch/CommandLineArgs.h

```diff
@@ -16,6 +16,7 @@ class CommandLineArgs
     const std::filesystem::path& ModelPath() const { return m_modelPath; }
     const std::string& HelpText() const { return m_helpText; }
     uint32_t DispatchIterations() const { return m_dispatchIterations; }
+    float TimeToRun() const { return m_timeToRun; }
```

Comment:
Perhaps something like `std::optional<float> TimeToRunInMilliseconds()` would make it clear that this is only intended to apply when it's explicitly set by the user (as opposed to a large default value).